include/sddf/blk/queue.h

```diff
@@ -49,6 +49,8 @@ typedef enum blk_resp_status {
     BLK_RESP_ERR_UNSPEC,
     /* invalid request parameters */
     BLK_RESP_ERR_INVALID_PARAM,
+    /* the device has been ejected */
```

Comment:
```suggestion
    /* the device is not connected */
```

would this be more accurate? Is `BLK_RESP_ERR_NO_DEVICE` possible if the card has never been inserted?